com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/DepthOfFieldCoCReproject.compute

```diff
@@ -59,14 +72,24 @@ void KMain(uint3 dispatchThreadId : SV_DispatchThreadID)
     // Sample the history buffer with the motion vector at the closest point
     float2 motionVector;
     DecodeMotionVector(LOAD_TEXTURE2D_X(_CameraMotionVectorsTexture, max(int2(posInputs.positionSS) + int2(closest.xy), 0)), motionVector);
-    float cocHis = SAMPLE_TEXTURE2D_X_LOD(_InputHistoryCoCTexture, sampler_LinearClamp, (uv - motionVector) * ScreenToTargetScaleHistory.xy, 0.0).x;
+    float cocHis = SAMPLE_TEXTURE2D_X_LOD(_InputHistoryCoCTexture, SAMPLER_STATE, (uv - motionVector) * ScreenToTargetScaleHistory.xy, 0.0).x;
 
     // Neighborhood clamping
+#ifdef ENABLE_MAX_BLENDING
+    float cocMin = closest.z;
+#else
     float cocMin = min(closest.z, 1.0f);
+#endif
     float cocMax = Max3(Max3(coc0, coc1, coc2), coc3, coc4);
     cocHis = clamp(cocHis, cocMin, cocMax);
 
     float outputCoC = lerp(coc0, cocHis, MotionBlending);
+#ifdef ENABLE_MAX_BLENDING
+    if (sign(coc0) != sign(cocHis))
```

Comment:
Nitpicking here but since it is used for the "fast" dof too: 
coc0 * cocHis < 0.0f  is faster (sign is 3 ops in itself) , so with mul is one vs 6 ops

Reply:
Indeed this is better, although this code path is not used in the fast DoF at all. 
(In theory it could improve some things there too, but I did not want to risk breaking old stuff)